ofl/notosansduployan/METADATA.pb

```diff
@@ -10,20 +10,26 @@ fonts {
   filename: "NotoSansDuployan-Regular.ttf"
   post_script_name: "NotoSansDuployan-Regular"
   full_name: "Noto Sans Duployan Regular"
-  copyright: "Copyright 2018 Google LLC. All Rights Reserved."
+  copyright: "Copyright 2018-2019 David Corbett. Copyright 2019-2022 Google LLC. All Rights Reserved."
+}
+fonts {
+  name: "Noto Sans Duployan"
+  style: "normal"
+  weight: 700
+  filename: "NotoSansDuployan-Bold.ttf"
+  post_script_name: "NotoSansDuployan-Bold"
+  full_name: "Noto Sans Duployan Bold"
+  copyright: "Copyright 2018-2019 David Corbett. Copyright 2019-2022 Google LLC. All Rights Reserved."
 }
-subsets: "menu"
 subsets: "duployan"
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/duployan.git"
+  archive_url: "https://github.com/notofonts/duployan/releases/download/NotoSansDuployan-v3.001/NotoSansDuployan-v3.001.zip"
+}
 is_noto: true
 languages: "chn_Dupl"  # Chinook Jargon, Duployan
 languages: "de_Dupl"  # German, Duployan
 languages: "fr_Dupl"  # French, Duployan
```

Comment:
If you’re going by the set of languages explicitly in scope for the Unicode proposal, you should remove German and add English, Lillooet, Okanagan, Romanian, Shuswap, and Thompson. You could also add Comox (Sliammon), Halkomelem, Latin, Sechelt, and Squamish, as they used essentially the same system as Chinook Jargon.

Reply:
I'll remove German for now. The others need lang repo magic.